js/scripts.js

```diff
@@ -48,6 +49,15 @@ $(function() {
 	    }
 	};
 
+	function Input() {
```

Comment:
this is simple function - so name of this functions should start with small character